api/openapi-spec/swagger.json

```diff
@@ -4321,9 +4321,10 @@
       "type": "string"
      },
      "volumeMode": {
-      "description": "volumeMode defines what type of volume is required by the claim. Value of Filesystem is implied when not included in claim spec.\n\nPossible enum values:\n - `\"Block\"` means the volume will not be formatted with a filesystem and will remain a raw block device.\n - `\"Filesystem\"` means the volume will be or is formatted with a filesystem.",
+      "description": "volumeMode defines what type of volume is required by the claim. Value of Filesystem is implied when not included in claim spec.\n\nPossible enum values:\n - `\"Auto\"` means the volume mode will be auto selected by CDI according to a matching StorageProfile\n - `\"Block\"` means the volume will not be formatted with a filesystem and will remain a raw block device.\n - `\"Filesystem\"` means the volume will be or is formatted with a filesystem.",
       "type": "string",
       "enum": [
+       "Auto",
```

Comment:
Interesting how this got picked up and included here.  I wonder if users could get confused thinking that "Auto" is a k8s core value?  Maybe should be a more CDI/StorageProfile specific name like `StorageProfileDefault`?

Just starting the discussion, not sure what is the best thing here

Reply:
Yep, `Auto` might be confusing, but not sure about `StorageProfileDefault`, as it's not default but function of the other PVC fields (`StorageClass`, `AccessMode`). Maybe `StorageProfileAuto`/`CDIAuto`? other suggestions?